src/runtime/tab/TabManager.ts

```diff
@@ -0,0 +1,50 @@
+import { HandlebarsTemplateContent } from 'src/api/HandlebarsTemplateContent';
+import { HandlebarsTab } from 'src/api/tab/HandlebarsTab';
+import { HtmlTab } from 'src/api/tab/HtmlTab';
+import type { CustomTabBase } from 'src/api/tab/CustomTabBase';
+import type { HtmlTabContent, Tab } from 'src/types/types';
+
+export class TabManager {
```

Comment:
Consider folding all the `ActorSheetRuntimeManager` content into `TabManager`. What else can be consolidated like this?

Reply:
Deferring to later. It's internal, so I have the leeway to do this out-of-band.